helpers.js

```diff
@@ -0,0 +1,22 @@
+exports.validMessage = function(message) {
```

Comment:
```suggestion
exports.validateMessage = function(message) {
```

Reply:
This function is related to the cities functionality, it is not as reusable function as `randomInt`, so put it in the other file

Reply:
Ok. I will do it

Reply:
Moved to file game.js